editor.html

```diff
@@ -207,19 +209,36 @@ <h2><i class="fa fa-unlock-alt"></i> <strong>{{ title }}</strong></h2>
+                          id="path4724" />
                         </svg>
                     </div>
                     <div class="roundlabel">Connect</div>
                 </a>
+                <a href="#" class="roundbutton action" id="command-connecting"
+                    tabindex="20"
```

Comment:
The tabindex needs to increase, also check the rest of the tab indexes to make sure they are in order.